tests/test_servable.py

```diff
@@ -1,89 +1,81 @@
-
-    return created_servable
-
-
-def test_servable_list_all():
-    cluster = create_test_cluster()
-    model = create_test_local_model()
-    upload_resp = model.upload(cluster)
-
-    Servable.create(model_name=upload_resp[model].modelversion.name,
-                    version=upload_resp[model].modelversion.version, cluster=cluster)
-
-    assert Servable.list(cluster=cluster)
-
-
-def test_servable_find_by_name():
-    pass
-
 
-def test_servable_list_for_modelversion():
-    pass
-
-
-def test_servable_delete():
-    cluster = create_test_cluster()
-    model = create_test_local_model()
-    ur = model.upload(cluster)
-
-    created_servable = Servable.create(model_name=ur[model].modelversion.name,
-                                       version=ur[model].modelversion.version, cluster=cluster,
-                                       metadata={"additionalProp1": "prop"})
-    time.sleep(1)
-
-    deleted_servable = Servable.delete(cluster, created_servable.name)
-
-    time.sleep(3)
-
-    with pytest.raises(ServableException):
-        found_servable = Servable.find(cluster, created_servable.name)
-
-
-def test_servable_create():
-    cluster = create_test_cluster()
-    model = create_test_local_model()
-    upload_resp = model.upload(cluster)
-
-    created_servable = Servable.create(model_name=upload_resp[model].modelversion.name,
-                                       version=upload_resp[model].modelversion.version, cluster=cluster)
-    found_servable = Servable.find(cluster, created_servable.name)
-
-    assert found_servable
-
-
-def test_servable_status():
-    cluster = create_test_cluster()
-    model = create_test_local_model()
-    upload_resp = model.upload(cluster)
-
-    created_servable = Servable.create(model_name=upload_resp[model].modelversion.name,
-                                       version=upload_resp[model].modelversion.version, cluster=cluster)
-
-    assert created_servable.status == ServableStatus.STARTING
-
-    time.sleep(10)
-    found_servable = Servable.find(cluster, created_servable.name)
+import pytest
+import sseclient
 
-    assert found_servable.status == ServableStatus.SERVING
+from hydrosdk.modelversion import LocalModel, ModelVersion
+from hydrosdk.exceptions import BadRequest
+from hydrosdk.servable import Servable, ServableStatus
+from tests.common_fixtures import *
+from tests.utils import *
+
+
+def test_servable_create(cluster: Cluster, local_model: LocalModel):
+    mv: ModelVersion = local_model.upload(cluster)
+    mv.lock_till_released()
+    sv: Servable = Servable.create(cluster, mv.name, mv.version)
+    assert Servable.find_by_name(cluster, sv.name)
+
+
+def test_servable_list_all(cluster: Cluster, local_model: LocalModel):
+    mv: ModelVersion = local_model.upload(cluster)
+    mv.lock_till_released()
+    sv: Servable = Servable.create(cluster, mv.name, mv.version)
+    assert sv.name in [servable.name for servable in Servable.list_all(cluster)]
```

Comment:
error messages in case of assert is wrong would be helpful

Reply:
@trthhrtz Would you mind suggesting changes while @tidylobster is on vacation?